b2luigi/batch/processes/htcondor.py

```diff
@@ -0,0 +1,226 @@
+        output = subprocess.check_output(cmd, env=self.task_env, cwd=submit_file_dir)
+        output = output.decode()
+        match = re.search(r"[0-9]+\.", output)
+        if not match:
+            raise RuntimeError("Batch submission failed with output " + output)
+
+        self._batch_job_id = int(match.group(0)[:-1])
+
+    def kill_job(self):
+        if not self._batch_job_id:
+            return
+        subprocess.check_call(["condor_rm", str(self._batch_job_id)], stdout=subprocess.DEVNULL)
+
+    def _create_htcondor_submit_file(self):
+        output_path = get_output_dirs(self.task, create_folder=True)
+        submit_file_path = os.path.join(output_path, "job.submit")
+        executable_wrapper_path = self._create_executable_wrapper()
+
+        log_file_dir = get_log_file_dir(self.task)
+        stderr_log_file = log_file_dir + "job.err"
+        stdout_log_file = log_file_dir + "job.out"
+        job_log_file = log_file_dir + "job.log"
+
+        general_settings = get_setting("htcondor_settings", dict())
+
+        try:
+            job_settings = self.task.htcondor_settings
+        except AttributeError:
+            job_settings = {}
+
+        with open(submit_file_path, "w") as submit_file:
+            submit_file.write(f"""
+            executable = {executable_wrapper_path}
+            should_transfer_files = YES
+            transfer_input_files = settings.json
+            log = {job_log_file}
+            output = {stdout_log_file}
+            error = {stderr_log_file}
+            """)
+
+            for key, item in general_settings.items():
+                submit_file.write(f"{key} = {item}\n")
+
+            for key, item in job_settings.items():
+                submit_file.write(f"{key} = {item}\n")
+
+            submit_file.write(f"queue 1\n")
+
+        return submit_file_path
+
+    def _create_executable_wrapper(self):
+
+        env_setup_path = get_setting("env_setup")
+
+        if not os.path.isfile(env_setup_path):
+            raise FileNotFoundError(f"Environment setup script {env_setup_path} does not exist.")
+
+        output_path = get_output_dirs(self.task, create_folder=True)
+        shell = get_setting("shell", "bash")
+
+        executable_wrapper_path = os.path.join(output_path, "executable_wrapper.sh")
+
+        condor_executable_cmd = " ".join(self.task_cmd)
+        with open(executable_wrapper_path, "w") as exec_wrapper:
+            exec_wrapper.write(f"#!/bin/{shell}\n")
+            exec_wrapper.write(f"source {env_setup_path}\n")
+            exec_wrapper.write("echo 'Starting executable'\n")
+            exec_wrapper.write("cd /jwd\n")
+            exec_wrapper.write(f"{condor_executable_cmd}\n")
+
+        # make wrapper executable
+        st = os.stat(executable_wrapper_path)
+        os.chmod(executable_wrapper_path, st.st_mode | stat.S_IEXEC)
+
+        return executable_wrapper_path
```

Comment:
I think you need to give the `abspath` of that, otherwise the default of just using "results" as the folder does not work.